cfgov/alerts/github_alert.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+import github3
+
+
+class GithubAlert(object):
+    def __init__(self, credentials):
+        self.token = credentials.get(
+            'token',
+            os.environ.get('GITHUB_TOKEN')
+        )
+        self.url = credentials.get(
+            'url',
+            os.environ.get('GITHUB_URL')
+        )
+        self.user = credentials.get(
+            'user',
+            os.environ.get('GITHUB_USER')
+        )
+        self.repo_name = credentials.get(
+            'repo_name',
+            os.environ.get('GITHUB_REPO')
+        )
+
+    def repo(self):
+        try:
+            gh = github3.login(
+                token=self.token,
+                url=self.url,
+            )
+            return gh.repository(self.user, self.repo_name)
+        except:
```

Comment:
[The Python docs](https://docs.python.org/2/howto/doanddont.html#except) recommend against using bare `except:` clauses like this because they can hide the real cause of errors and also prevent breaking code using e.g. Ctrl-C. Would it be sufficient to catch [`GithubError`](https://github.com/github3py/github3py/blob/master/github3/models.py#L357)?

Reply:
Oh nice, good catch!

Reply:
Thinking about this more, I'm going to remove the `try` entirely.  The Github API will throw its own (more specific) errors.